File: Go/BEGINNER/1008/1008.go
package main

import "fmt"

func main() {

	var number int
	var workedHours, receivePerHour float64

	fmt.Scanln(&number)
	fmt.Scanln(&workedHours)
	fmt.Scanln(&receivePerHour)

	salary := workedHours * receivePerHour

	fmt.Println("NUMBER =", number)
	fmt.Printf("SALARY = U$ %.2f\n", salary)
}

File: C Sharp/UriOnlineJudge/BEGINNER/_1008.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace UriOnlineJudge.BEGINNER
{
    class _1008
    {
        public void Execute()
        {
            int number, workedHours;
            double receivePerHour;

            number = int.Parse(Console.ReadLine());
            workedHours = int.Parse(Console.ReadLine());
            receivePerHour = double.Parse(Console.ReadLine());

            var salary = workedHours * receivePerHour;

            Console.WriteLine("NUMBER = " + number + "\n" +
                "SALARY = U$ " + salary.ToString("000.00"));

        }
    }
}
